addons/bus/static/src/services/bus_service.js

```diff
@@ -92,7 +93,7 @@ export const busService = {
             // Let's format it in order to ease its usage:
             //     - number if user is logged, false otherwise, keep
             //       undefined to indicate session_info is not available.
-            let uid = Array.isArray(session.user_id) ? session.user_id[0] : session.user_id;
+            let uid = user.userId;
```

Comment:
You might want to update the above comment if this is no longer relevant, as well as the code below.
But is the PR actually making sure the user is now consistent across all contexts?
cc @tsm-odoo

Reply:
Ok actually this one shouldn't have been replaced. In session_info, in the backend, we have user_id and uid, which are basically the same. In the frontend, I see we don't have uid. I'll try to uniformize this, maybe in another PR. For now, we can keep the code as it was as my PR doesn't delete user_id from the session, so it should keep working as expected.